output/transform/transform_test.go

```diff
@@ -66,189 +68,198 @@ func TestStatements(t *testing.T) {
 		ServerStatistic: &pganalyze_collector.ServerStatistic{},
 		Replication:     &pganalyze_collector.Replication{},
 		QueryReferences: []*pganalyze_collector.QueryReference{
-			&pganalyze_collector.QueryReference{
```

Comment:
This is unrelated, but element types can be inferred when the containing array type is specified, so including the type here is unnecessary.